operators/multiclusterobservability/manifests/base/config/metrics_allowlist.yaml

```diff
@@ -193,9 +194,6 @@ data:
           - __name__="kube_pod_container_resource_limits",namespace="{{ $labels.namespace }}"
           - __name__="kube_pod_container_resource_requests",namespace="{{ $labels.namespace }}"
           - __name__="kube_resourcequota",namespace="{{ $labels.namespace }}"
-          - __name__="mixin_pod_workload",namespace="{{ $labels.namespace }}"
-          - __name__="namespace_workload_pod:kube_pod_owner:relabel",namespace="{{ $labels.namespace }}"
```

Comment:
Removed recording rules from the collection rules list. These will still be captured within the default metrics list.